### PR TITLE
chore: Update dependencies android

### PR DESCRIPTION
**Pull request by bensonarafat**
Issue #670

### CODE REVIEW

**Comment by fbernaly** [comment]
- test with example app
- update versions
- update changelog

**Comment by bensonarafat** [comment]
> * test with example app
> * update versions
> * update changelog

Thanks, I've updated the changeLog and versions 
Also, test the example app and the various packages are working

**Comment by fbernaly** [comment]
@bensonarafat : you are missing to update the CHANGELOG in every package, for example:

https://github.com/flutter-ml/google_ml_kit_flutter/blob/develop/packages/google_mlkit_barcode_scanning/CHANGELOG.md

**Comment by fbernaly** [comment]
What about face mesh, digital ink recog, subject seg, doc scanner, and entity extraction? do they have new dependencies?

**Comment by bensonarafat** [comment]
@fbernaly 
No dependencies for those.

Oh, i didn't know about the changelog for each package. 
But why did you close it?

**Comment by fbernaly** [comment]
Oh, sorry, I think I close it by mistake.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,65 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2024-08-29
```
Comment (fbernaly) [inline CHANGELOG.md:6]: change the date to 2024-09-04

Comment (bensonarafat) [reply CHANGELOG.md:6]: Alright

**Comment by bensonarafat** [comment]
if I can't correct this night. i will correct tomorrow @fbernaly

**Comment by bensonarafat** [comment]
@fbernaly 
This is been updated. 
Other packages that were not included, there are no new updates. 
Working on the subject segmentation

**Comment by cto-seak-global** [comment]
@fbernaly Do you have an update on this? I haven't been able to find a set of downgrades that are compatible with my app, so my release is blocked until this issue is fixed.

**Comment by FantaMagier** [comment]
Any news on that? I need the newest version for the text reco ml kit

**Review by fbernaly — APPROVED** [review]

**Comment by FantaMagier** [comment]
@bensonarafat When can we expect the release on Pub Dev? Thanks for the work.

**Comment by bensonarafat** [comment]
@FantaMagier 
@fbernaly should release a new update on pub.dev soon. 
meanwhile, you can use this branch directly on your pubspec.dev 
eg. 

```
  google_ml_kit:
    git:
      url: https://github.com/flutter-ml/google_ml_kit_flutter.git
      ref: develop
```

**Pull request merged**